odc/geo/_xr_interop.py

```diff
@@ -245,6 +245,59 @@ def assign_crs(
     return xx
```

Comment:
pylint complains about too-may-lines, add disable line for that at the top of the file
`#pylint: disable=<message-name>`